### DISCUSSION TITLE
启动comfyUI服务,blender输入节点就消失

### DISCUSSION THREAD

**Discussion by bengenseo** [AIGODLIKE/ComfyUI-BlenderAI-node#88] [General]
![1232](https://github.com/AIGODLIKE/ComfyUI-BlenderAI-node/assets/34401610/5ae1b93c-b0d1-4587-8304-2b1b10837b07)
![2024-05](https://github.com/AIGODLIKE/ComfyUI-BlenderAI-node/assets/34401610/de438bc5-6118-4b4f-b4b0-3ad2554dab2c)

**Comment by genespeed1** [comment]
这是作者的一个bug，blender AI node里CUP node的文件夹他忘记放东西进去了。。。导致每次blender和comfy通讯后，comfy里的CUP内容会同步blender AI node，直接清空。
方法如下：把comfy custom nodes/ComfyUI-CUP夹子里的东西全部复制到blender插件文件夹的同名目录下（C:\Users\你的用户名\AppData\Roaming\Blender Foundation\Blender\你的版本号\scripts\addons\ComfyUI-BlenderAI-node-main\SDNode\custom_nodes\ComfyUI-CUP，完美解决
